webpack/assets/javascripts/react_app/components/users/PersonalAccessTokens/PersonalAccessTokensReducer.js

```diff
@@ -0,0 +1,34 @@
+import Immutable from 'seamless-immutable';
+import {
+  PERSONAL_ACCESS_TOKENS_REQUEST,
+  PERSONAL_ACCESS_TOKENS_SUCCESS,
+  PERSONAL_ACCESS_TOKENS_FAILURE,
+  PERSONAL_ACCESS_TOKEN_FORM_SUBMITTED,
+  PERSONAL_ACCESS_TOKEN_CLEAR,
+} from './PersonalAccessTokensConstants';
+
+const initialState = Immutable({ tokens: [] });
+
+const sortTokens = tokens => tokens.sort((a, b) => a.created_at < b.created_at);
```

Comment:
any data manipulation should be done through selectors.
selectors can be memorized while reducers should act fast